aries-site/src/examples/components/pagination/PaginationServerTableExample.js

```diff
@@ -53,13 +62,16 @@ const columns = [
 ];
 
 export const PaginationServerTableExample = () => {
+  const size = useContext(ResponsiveContext);
   const [sort, setSort] = useState({ property: 'name', direction: 'asc' });
   const [data, setData] = useState([]);
   const [numberItems, setNumberItems] = useState(0);
   const [page, setPage] = useState(1);
-  const limit = 10;
+  const [limit, setLimit] = useState(10);
   const pageResultStart = (page - 1) * limit + 1;
   const pageResultEnd = Math.min(page * limit, numberItems);
+  const options = [5, 10, 25, 50];
```

Comment:
We can pull this `options` variable outside of the PaginationServerTableExample since it's not reliant on anything in here. (just placing on the line before, like "columns" is fine.